test/node/features/restify.feature

```diff
@@ -1,4 +1,4 @@
-@skip_node_18
```

Comment:
18 was failing for the same reason as 20 (restify needed bumping) and it had been disabled a while back with the intention of coming back and fixing it, but that was forgotten about until now